app/views/hosts/_list.html.erb

```diff
@@ -36,6 +36,18 @@
     <% end %>
   </tbody>
 </table>
+<div id="selected_hosts">
+  <table class="table-condensed">
+    <tr>
+      <td class="ca">
+        <%=check_box_tag "keep_selection", "", false, { :title => _("Keep Selected hosts for a future action")} %>
```

Comment:
Missing space after `=`, no need for braces around the hash and "Selected" should be lower case.